### DISCUSSION TITLE
Multiplayer / Items

### DISCUSSION THREAD

**Discussion by RomainCoste1** [DaXcess/LCVR#13] [Q&A]
Hey, i'm really interested in your work and i too thought this game needed Vr so thank you for that, i do have a couple of questions tho: 
1. How is multiplayer fonctionnal currently is it just a on and off Vr setting with steam and how will it look with others
2. Are items usable yet like jetpack / terminal / doors...
3. How is the project getting along and what advancement have you made until now
"sorry for the barrage of questions im just really hyped X)"

**Comment by AbandonedPlayer** [comment]
There is ongoing discussions about multiplayer and progress on this build on discord.
https://flat2vr.com/
https://discord.com/channels/747967102895390741/1173840916788809779/1189878630512332811
